File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/PoengarManuell.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.AvtalelandCti;

import java.io.Serializable;
import java.util.Date;

/**
 * Klassen beskriver et poengår opptjent i utlandet. Settes av saksbehandler.
 */
public class PoengarManuell implements Serializable {


    /**
     * Poengåret fra og med dato.
     */
    private Date fom;

    /**
     * Poengåret til og med dato.
     */
    private Date tom;

    /**
     * Angir om poengåret skal brukes i pro rata beregning.
     */
    private boolean ikkeProrata;

    /**
     * Angir om poengåret skal brukes i alternativ pro rata beregning.
     */
    private boolean ikkeAlternativProrata;

    /**
     * Avtaleland som poengår ble opptjent i.
     */
    private AvtalelandCti avtaleland;

}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/PoengarManuell.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.kode.AvtalelandCti
import java.io.Serializable
import java.util.*

/**
 * Klassen beskriver et poengår opptjent i utlandet. Settes av saksbehandler.
 */
class PoengarManuell : Serializable {
    /**
     * Poengåret fra og med dato.
     */
    var fom: Date? = null

    /**
     * Poengåret til og med dato.
     */
    var tom: Date? = null

    /**
     * Angir om poengåret skal brukes i pro rata beregning.
     */
    var isIkkeProrata = false

    /**
     * Angir om poengåret skal brukes i alternativ pro rata beregning.
     */
    var isIkkeAlternativProrata = false

    /**
     * Avtaleland som poengår ble opptjent i.
     */
    var avtaleland: AvtalelandCti? = null
}